packages/server/src/__tests__/ApolloServer.test.ts

```diff
@@ -8,6 +8,7 @@ import {
 } from '../plugin';
 import { makeExecutableSchema } from '@graphql-tools/schema';
 import { HeaderMap } from '../runHttpQuery';
+import { mockLogger } from './logger.test';
```

Comment:
I feel a little weird about test files importing from each other, just because I don't really understand jest well enough to feel like this doesn't result in the tests being defined in multiple test suites? can this go into a non `.test.ts` file?

Reply:
[11a8570](https://github.com/apollographql/apollo-server/pull/6518/commits/11a85701a2d2e5f667cc8a783b7a199c0e1cad6a)